### ISSUE TITLE
Url not being passed to app

### ISSUE THREAD

**Issue by DanielSmith1239** [tomhamming/ControlCenterOpenApp#1]
Hello,

I'm experimenting with adding similar functionality to my app using your example, and it looks like the url is never being handled by the app. It seems to just be opening the app because `openAppWhenRun` is set to `true`. Is this accurate? Am I doing something wrong?

**Comment by tomhamming** [comment]
Yeah, the reason this example project exists is because the app never gets to handle the URL. That's why it's writing something to `UserDefaults`, to demonstrate passing data that way.

A more full example would be to put the URL in `UserDefaults` instead of a random bit of text, and check `UserDefaults` for a URL to handle on app resume and launch.

**Comment by DanielSmith1239** [comment]
In that case, why use an AppOpensIntent?

**Comment by tomhamming** [comment]
I got this as advice from Brian Mueller on Mastodon [here](https://mastodon.social/@brianmueller/113154977260389354#.) - it's the only way I know so far to accomplish opening the app to a particular screen from a Control Center widget. If you have other suggestions and can provide a better working example, please feel free to fork this repo and add another example. :-)

**Comment by DanielSmith1239** [comment]
All good, just trying to figure out the best way to do things. There isn't a ton of documentation on this yet.
